### ISSUE TITLE
Where is the Catalan branch?

### ISSUE THREAD

**Issue by masipcat** [AnySoftKeyboard/LanguagePack#162]
I'm opening this issue because I found some bugs on the Catalan keyboard that makes the keyboard quit unusable. If I write "Hola" + space the auto-suggestion replaces "Hola" with "Vila".

![bug](https://user-images.githubusercontent.com/775189/42735976-fc6c0346-885d-11e8-8a00-105f6481449a.png)

I would like to know where is the code for Catalan language pack and try to figure out why the keyboard is suggesting those words.

Thank you,

**Comment by xmiseggs** [comment]
Enclosed here is my post about the Catalan and the problem, thank you I hope you will solve it:
https://github.com/AnySoftKeyboard/AnySoftKeyboard/issues/1372

**Comment by friesenkiwi** [comment]
Thank you two for your reports! I cannot quite say what the problem here might be.
Catalan is one of the few packs we, until now, unfortunately did not manage to upheave to the modern build setup in Android Studio/Gradle but was buried in the old SVN sources of Google code (Eclipse style). I just added a new branch (https://github.com/AnySoftKeyboard/LanguagePack/tree/Catalan) with those old files so you can investigate. Currently I am not able to tell you how to build etc. but maybe it already helps to debug the problem? Otherwise, we'd need to update to modern style first.

@xmiseggs I think you can close https://github.com/AnySoftKeyboard/AnySoftKeyboard/issues/1372 over at the mother project then, so as we don't have duplicates :)

**Comment by xmiseggs** [comment]
@friesenkiwi Closed. Thank you very much.

**Comment by masipcat** [comment]
@friesenkiwi Thank you very much for the answer and publishing the code. I'll start trying to migrate to Android Studio

**Comment by masipcat** [comment]
After playing with the new version of the catalan keyboard I still face the problem I described in that issue:

![index-crop](https://user-images.githubusercontent.com/775189/44395401-14c5e580-a53a-11e8-80f3-09c4f67785a0.jpeg)

Now if I write "Captura" the keyboard suggests me "Fractura". Both words are correct but they have different meaning. I checked the dictionary I found:

```
 <w f="100">factura</w>
 <w f="82">captura</w>
```

Any thoughts on how can I prevent that from happening? @friesenkiwi
